alerts/cluster-monitoring-operator/NodeClockNotSynchronising.md

```diff
@@ -2,66 +2,139 @@
 
 ## Meaning
 
-The `NodeClockNotSynchronising` alert triggers when a node is affected by
-issues with the NTP server for that node. For example, this alert might trigger
-when certificates are rotated for the API Server on a node, and the
-certificates fail validation because of an invalid time.
+The `NodeClockNotSynchronising` alert triggers when an OpenShift worker node (RHCOS) is unable to synchronize its clock with configured time servers. This indicates a critical time synchronization failure that, if left unaddressed, can lead to serious cluster issues.
 
+The alert is specifically triggered when:
+- The Prometheus metric `node_timex_sync_status` returns `0` (indicating NTP is not synchronizing)
+- The metric `node_timex_maxerror_seconds` is greater than or equal to `16` seconds
```

Comment:
we don't usually put details about the alert expression because this might change over time and it's impossible to keep the documentation in sync.